lib/rules/selector-pseudo-element-case/__tests__/index.js

```diff
@@ -6,6 +6,7 @@ const { messages, ruleName } = rule;
 testRule(rule, {
   ruleName,
   config: ["lower"],
+  fix: true,
```

Comment:
Could you add a test case with comments in the selector?

Reply:
Unfortunately it won't work until update to the last version of `postcss-selector-parser` (#3284). 
I don't sure that it makes sense to add some workaround.

Reply:
In [selectorCombinatorSpaceChecker.js](https://github.com/stylelint/stylelint/blob/master/lib/rules/selectorCombinatorSpaceChecker.js#L14), it choose the selector to pass to `parseSelector` beforehand.

Reply:
I added few new test cases with comments inside selectors.